hosts/ponkila-ephemeral-gamma/nix-store-query.txt

```diff
@@ -2147,7 +2145,6 @@
 /nix/store/977dhhb4yd6jqldb0yybzf7fbp24xfiy-unit-systemd-zram-setup-.service.drv
 /nix/store/asxzcpcmpgmvhl04cy18js32f9x9pyi0-unit-user-.service.drv
 /nix/store/k9g00s0miyz8819b7wsa3xpxx9428ch3-unit-user-runtime-dir-.service.drv
-/nix/store/blx4jrf62kvl74cijqy2ph9wym61ynvb-unit-wg0.service.drv
```

Comment:
Notice that this also witnesses a regression on gamma: it needs to have its configuration updated.

Reply:
This is because https://github.com/ponkila/nixobolus/pull/13 introduces the `homestakeros.wireguard.enable` option, which disables WireGuard for ponkila-gamma. We should only enable it if needed, and currently gamma does not even have wg0.conf.